.ci/build_script.sh

```diff
@@ -13,12 +13,12 @@ if [ "$TARGET" = kobo ]; then
         -v "${HOME}/.ccache:${DOCKER_HOME}/.ccache" \
         -v "$(pwd):${DOCKER_HOME}/base" "${DOCKER_IMG}" \
         /bin/bash -c 'source /home/ko/.bashrc && cd /home/ko/base && sudo chown -R ko:ko . && make TARGET=kobo all'
-elif [ "$TARGET" = kindle ]; then
+elif [ "$TARGET" = kindle5 ]; then
     sudo chmod -R 777 "${HOME}/.ccache"
     docker run -t \
         -v "${HOME}/.ccache:${DOCKER_HOME}/.ccache" \
         -v "$(pwd):${DOCKER_HOME}/base" "${DOCKER_IMG}" \
-        /bin/bash -c 'source /home/ko/.bashrc && cd /home/ko/base && sudo chown -R ko:ko . && make TARGET=kindle5 all'
+        /bin/bash -c 'source /home/ko/.bashrc && cd /home/ko/base && sudo chown -R ko:ko . && make TARGET=kindle5 all; echo $PATH'
```

Comment:
Wouldn't it make more sense to add the debug at the beginning so you don't have to wait half an hour? :-P

Output:
```
/home/ko/x-tools/arm-kobo-linux-gnueabihf/bin:/home/ko/x-tools/arm-kindlepw2-linux-gnueabi/bin:/home/ko/x-tools/arm-kindle5-linux-gnueabi/bin:/home/ko/x-tools/arm-kindle-linux-gnueabi/bin:/home/ko/.luarocks/bin:/home/ko/local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
```

Same Docker image output with TARGET=kindle5 from the nightly build here (with current base, obviously):
https://gitlab.com/koreader/nightly-builds/-/jobs/64268539

Don't really have time to look into this closer but maybe it'll help.